Add Article "From identification to "rawification": what does opening government data mean?"

samgoeta opened the pull request.

samgoeta committed:
Create from-identification-to-rawification-what-does-opening-government-data-means

samgoeta committed:
Update from-identification-to-rawification-what-does-opening-government-data-means

samgoeta committed:
Update from-identification-to-rawification-what-does-opening-government-data-means

samgoeta committed:
Merge pull request #1 from samgoeta/patch-1

Merge gh-pages and patch-1

morchickit commented on guide/en/how-to-open-up-data/from-identification-to-rawification-what-does-opening-government-data-means:
```diff
@@ -0,0 +1,53 @@
+###Introduction
+Open government data has been at the center of several transparency policies and has been established as a legal obligation in many countries. Debates and discussions about open data have mainly focused on its opportunities and its capacity to develop transparency and innovation. Yet, how government data are concretely “opened” remain largely overlooked, as if data was, in its very nature, an eas ...
+Rather, an exploration of the backrooms of open government data projects in several French administrations [1] shows that openness, far from being an intrinsic property of data, is the result of a specific work. Three main kinds of operations can be identified, which punctuate such work and lead to the production of open government data: identification, extraction and "rawification". They all sho ...
+
```
Can we add here to whom this data is useful? I think that if we will add a line or two that will explain that this data is beneficial for civil society and why they should read about the back rooms of governments, it will go with the narrative of the whole book.

morchickit commented on guide/en/how-to-open-up-data/from-identification-to-rawification-what-does-opening-government-data-means:
```diff
@@ -0,0 +1,53 @@
+Open government data has been at the center of several transparency policies and has been established as a legal obligation in many countries. Debates and discussions about open data have mainly focused on its opportunities and its capacity to develop transparency and innovation. Yet, how government data are concretely “opened” remain largely overlooked, as if data was, in its very nature, an eas ...
+Rather, an exploration of the backrooms of open government data projects in several French administrations [1] shows that openness, far from being an intrinsic property of data, is the result of a specific work. Three main kinds of operations can be identified, which punctuate such work and lead to the production of open government data: identification, extraction and "rawification". They all sho ...
+
+###Identification
```
Can you add one line about the meaning of identification? I assume is the identification of datasets that need to be released, but I think that it will help just to make it clear to the reader.

samgoeta committed:
Update from-identification-to-rawification-what-does-opening-government-data-means

As suggested by @morchickit, Add one line in intro on potential readers + one line on identification
